### PR TITLE
Small fixes to the UI and making it unable to send blank message

### PR DESCRIPTION
**Pull request by sibela46**

### CODE REVIEW

**Review by iatkins — DISMISSED** [review]

File: server/models/GameLogic.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const promptDuration = process.env.NODE_ENV === 'testing' ? 4000 : 30000;
 const voteDuration = process.env.NODE_ENV === 'testing' ? 5000 : 30000;
 const storytellerDuration = process.env.NODE_ENV === 'testing' ? 4000 : 60000;
 const nextRoundDuration = process.env.NODE_ENV === 'testing' ? 2000 : 5000;
-const minPlayers = process.env.NODE_ENV === 'testing' ? 2 : 3;
+const minPlayers = process.env.NODE_ENV === 'testing' ? 2 : 1;
```
Comment (tillywoodfield) [inline server/models/GameLogic.js]: hmm...

File: server/services/socket.js [hunk 2]
```diff
@@ -82,7 +82,10 @@ exports.emitPlayers = (roomId, players) => sockets.forEach(socket => socket.hand
 exports.emitNewRound = (roomId, status, roundNum, currentPlayer, timeoutDuration) => sockets.forEach(socket => {
     if (socket.handshake.session.roomId === roomId) {
         socket.emit("new round", { status, roundNum, currentPlayer });
+        console.log(socket.handshake.session.user);
         if (socket.handshake.session.user.username === currentPlayer.username) {
+            console.log("Timeout");
```
Comment (tillywoodfield) [inline server/services/socket.js]: logs

File: src/components/Lobby/Lobby.module.css [hunk 3]
```diff
@@ -1,21 +1,22 @@
 .main {
     display: inline-block;
-    height: 100vh;
+    height: 100%;
     width: 100%;
     align-items: stretch;
```
Comment (tillywoodfield) [inline src/components/Lobby/Lobby.module.css:5]: This line doesn't do anything

File: src/components/Lobby/Lobby.module.css [hunk 4]
```diff
@@ -1,21 +1,22 @@
...
 }
 
 .header {
+    height: 15%;
```
Comment (tillywoodfield) [inline src/components/Lobby/Lobby.module.css:9]: Have you checked that this doesn't size weirdly as you resize the page?

File: src/components/Lobby/Room/Room.jsx [hunk 5]
```diff
@@ -10,15 +10,15 @@ export class Room extends React.Component {
         const waitingVisible = !this.props.room.started && (this.props.room.minPlayers - this.props.room.players.length > 0);
         return (
             <div className={styles.room} key={this.props.room.roomId} data-cy="room">
-                <h2 data-cy="room-title">{"Room: " + this.props.room.roomId}</h2>
-                <ul id="players" data-cy='room-players'>
-                    {this.props.room.players.map((player, key) => <li key={key}><span data-cy='player-username'>{player.username}</span></li>)}
+                <h2 className={styles.roomHeader} data-cy="room-title" >{"Room: " + this.props.room.roomId}</h2>
```
Comment (tillywoodfield) [inline src/components/Lobby/Room/Room.jsx]: You've added a random space after room title

Comment (sibela46) [reply src/components/Lobby/Room/Room.jsx]: oopsie woopsie

**Review by tillywoodfield — APPROVED** [review]

**Pull request merged**